### PR TITLE
Sunset rebrandly

### PR DESCRIPTION
**Pull request by YurkoWasHere**
Removed circleci tasks for rebrandly

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,15 +10,9 @@ workflows:
   # See: https://ideas.circleci.com/ideas/CCI-I-627
   on-commit:
     jobs:
```
Comment (benhylau) [inline .circleci/config.yml]: I don't think we can leave this open? Not sure.

```suggestion
#  on-commit:
#    jobs:
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -10,15 +10,9 @@ workflows:
...
-      - update-shortlinks:
-          filters:
-            branches:
-              only: master
-
   # scheduled workflow
   every-15-min:
```
Comment (benhylau) [inline .circleci/config.yml]: Probably this whole block needs to be commented out since there is no `job`.

**Review by benhylau — CHANGES_REQUESTED** [review]
Please also update README.md references to this job. Make sure it reflects reality.

File: .circleci/config.yml (hunk 2 above)
Comment (YurkoWasHere) [reply .circleci/config.yml]: removed

**Comment by YurkoWasHere** [comment]
Not sure if this is correct. 
Error gone away.

**Review by benhylau — APPROVED** [review]

**Pull request merged**